sim/src/tlv.rs

```diff
@@ -385,9 +404,14 @@ impl ManifestGen for TlvGen {
             estimate += 4 + if aes256 { 96 } else { 80 };
         }
 
-        // Gather the size of the dependency information.
+        // Gather the size of the dependency information and security counters.
         if self.protect_size() > 0 {
```

Comment:
Could we use the return value of `protect_size()` directly? Just like in `make_tlv()`?